tutorials/contour/index.pug

```diff
@@ -0,0 +1,69 @@
+extends ../common/index.pug
+
+block mainTutorial
+  :markdown-it
+    # Tutorial - Contour map
+    First, let's create our map and add a base map and a feature layer.
+  
+  +codeblock('javascript', 1).
+    var map = geo.map({
+      node: '#map',
+      center: { x: -157.965, y: 21.482 },
+      zoom: 10
+    });
+    map.createLayer('osm');
+    var layer = map.createLayer('feature');
+
+  :markdown-it
+    Then load some data. 
+    The data is grid data describing geospatial point elevation, a record of z value -9999 means it's sea level. 
+    Since the data is relatively big, intead of adding inline data, we load asynchronously, and use [promise chain](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises) to chain up the following steps.
+  +codeblock('javascript', 2, 1).
+    var promise = Promise.resolve($.ajax({
+      url: '../../data/oahu.json'
+    }));
+
+  :markdown-it
+    Once the data is loaded. We create the contour feature with default color range. *min: 0* means only showing data with elevation.
```

Comment:
You can surround examples that are in the code with backticks (rather than using an asterisk to italicize it).  With the change the background color for the tutorials, we may want to add `code: { background: <some contrasting color>; }` to the tutorial css.